src/utilities/include/metaphysicl/std_array_wrapper.h

```diff
@@ -0,0 +1,97 @@
+//-----------------------------------------------------------------------bl-
+//--------------------------------------------------------------------------
+//
+// MetaPhysicL - A metaprogramming library for physics calculations
+//
+// Copyright (C) 2013 The PECOS Development Team
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the Version 2.1 GNU Lesser General
+// Public License as published by the Free Software Foundation.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc. 51 Franklin Street, Fifth Floor,
+// Boston, MA  02110-1301  USA
+//
+//-----------------------------------------------------------------------el-
+
+#ifndef METAPHYSICL_STD_ARRAY_WRAPPER_H
+#define METAPHYSICL_STD_ARRAY_WRAPPER_H
+
+#include "metaphysicl/metaphysicl_asserts.h"
+
+#include <array>
+
+namespace MetaPhysicL
+{
+template <typename T, size_t N>
+struct DynamicStdArrayWrapper
```

Comment:
What's up with DynamicStdArrayWrapper in this file vs in the more name-matching file?

Reply:
Lol, I guess I somehow duplicated the file at some point. Will remove